### PR TITLE
Add method `addMember` to Conference model. 

### PR DESCRIPTION
**Pull request by a-type**
Adds a call as a member of a conference.

### CODE REVIEW

**Comment by fschmager** [comment]
The overall code of this Java SDK is so terrible. The added code seems to be following those "conventions". Not much I can say without ripping the whole SDK apart.

**Comment by a-type** [comment]
Agreed. The C# SDK has similar problems. I don't know why we are using Maps of strings and objects in these languages. But for now, it's just status quo, in order to add this feature to unblock Nick and myself.

**Comment by dpereira-daitan** [comment]
Thanks for removing those unnecessary finals.
Approved.

File: src/test/java/com/bandwidth/sdk/model/ConferenceTest.java [hunk 1]
```diff
@@ -166,7 +167,47 @@ public void shouldGetMembers() throws Exception {
...
+                "}");
+
+        Conference conference = new Conference(mockClient, jsonObject);
+
+        Map<String, Object> memberOptions = new HashMap<String, Object>();
+        memberOptions.put("callId", "{callId}");
+
+        ConferenceMember member = conference.addMember(memberOptions);
+        assertThat(member.getCall(), equalTo("https://localhost:8444/v1/users/{userId}/calls/{callId}"));
+        assertThat(mockClient.requests.get(0).name, equalTo("post"));
```
Comment (fschmager) [inline src/test/java/com/bandwidth/sdk/model/ConferenceTest.java]: extract local variable see [earlier comment](https://github.com/bandwidthcom/java-bandwidth/pull/33#discussion_r51028302)

File: src/main/java/com/bandwidth/sdk/model/Conference.java [hunk 2]
```diff
@@ -177,6 +177,24 @@ public void mute() throws Exception {
...
+     * Adds a call to the conference as a conference member. The call must be created with the conference ID.
+     *
+     * @param params parameter map for the request. see http://ap.bandwidth.com/docs/rest-api/conferences/#resourcePOSTv1usersuserIdconferencesconferenceIdmembers
+     * @return the model for the conference member
+     * @throws Exception
+     */
+    public ConferenceMember addMember(Map<String, Object> params) throws Exception {
+        String membersPath = StringUtils.join(new String[]{
+                getUri(),
+                "members"
```
Comment (fschmager) [inline src/main/java/com/bandwidth/sdk/model/Conference.java]: magic string

File: src/main/java/com/bandwidth/sdk/model/Conference.java [hunk 3]
```diff
@@ -177,6 +177,24 @@ public void mute() throws Exception {
...
+        }, '/');
```
Comment (fschmager) [inline src/main/java/com/bandwidth/sdk/model/Conference.java]: magic string

File: src/main/java/com/bandwidth/sdk/model/Conference.java [hunk 4]
```diff
@@ -177,6 +177,24 @@ public void mute() throws Exception {
...
+
+        RestResponse response = client.post(membersPath, params);
+        JSONObject memberObj = toJSONObject(client.get(response.getLocation(), null));
```
Comment (fschmager) [inline src/main/java/com/bandwidth/sdk/model/Conference.java]: magic `null`

**Pull request closed without merging**